Game/Particles/Particle.cs

```diff
@@ -0,0 +1,36 @@
+﻿using MainGame.Players.PlayerStates;
+using MainGame.SpriteHandlers;
+using MainGame.SpriteHandlers.ParticleSprites;
+using Microsoft.Xna.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace MainGame.Particles
+{
+    public class Particle : IParticle
+    {
+        public ISprite Part;
+        private readonly Game1 game;
+        public Particle(Game1 game)
+        {
+          
+           
+            Part = SpriteFactory.CreateSpawnParticles();
+            this.game = game;
```

Comment:
I don't think Game1 is a necessary parameter for this class. I see you used it to determine the position. In the future, particles spawn position will actually be based on enemies' position instead. You could have a position be passed instead.